src/app/subnets/components/ReservedRanges/ReservedRanges.tsx

```diff
@@ -270,26 +143,20 @@ const ReservedRanges = ({
           No subnets are available on this VLAN. Ranges cannot be reserved.
         </Notification>
       ) : null}
-      <MainTable
+      <GenericTable
         className={classNames(
           "reserved-ranges-table",
           "p-table-expanding--light",
           {
             "reserved-ranges-table--has-subnet": showSubnetColumn,
           }
         )}
-        defaultSort="name"
-        defaultSortDirection="descending"
-        emptyStateMsg={generateEmptyStateMsg(tableStatus, {
-          default: `No IP ranges have been reserved for this ${
-            isSubnet ? "subnet" : "VLAN"
-          }.`,
-        })}
-        expanding
-        headers={headers}
-        responsive
-        rows={generateRows(ipRanges, showSubnetColumn, setSidePanelContent)}
-        sortable
+        columns={columns}
+        data={data}
+        isLoading={ipRangeLoading}
+        noData={`No IP ranges have been reserved for this ${isSubnet ? "subnet" : "VLAN"}.`}
+        role="table"
+        sortBy={[{ id: "name", desc: true }]}
```

Comment:
The sorting `id` must match whatever column you are sorting by. This table doesn't have a `"name"` column. Use ascending startIp to display the IPs in sequence.
```suggestion
        sortBy={[{ id: "startIp", desc: false }]}
```